src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -47,6 +48,9 @@
   --#{$menu-toggle}--disabled--Color: var(--pf-t--global--text--color--on-disabled);
   --#{$menu-toggle}--disabled--BackgroundColor: var(--pf-t--global--background--color--disabled--default);
 
+  // * Menu toggle icon
+  --#{$menu-toggle}__toggle-icon--Height: calc(var(--#{$menu}__item--FontSize) * var(--#{$menu}__item--LineHeight));
```

Comment:
Just want to confirm, should this be for the menu-toggle__toggle-icon (down caret) or the menu-toggle__icon (whatever is wrapped in the `pf-v6-c-menu-toggle__icon` before/instead of the toggle text)? It works regardless (assuming part of the intent was to fix the height of a plain toggle that had the small modifier).

Reply:
Good observation, added the same concept for `toggle__icon`